homeassistant/components/velux/entity.py

```diff
@@ -15,25 +16,26 @@ class VeluxEntity(Entity):
     _attr_should_poll = False
     _attr_has_entity_name = True
 
-    def __init__(self, node: Node, config_entry_id: str) -> None:
+    def __init__(self, node: Node, config_entry: VeluxConfigEntry) -> None:
```

Comment:
I personally don't really enjoy passing the config_entry everywhere as it feels like the easy option, I think we can still do with just the entry id

Reply:
I prefer having it consistent throughout the integration but I'm not insisting, let me know if you want it changed.

Reply:
In the sense that all initializers of Entity and derived classes use VeluxConfigEntry.

Reply:
Finally, that took me a while to understand what you meant, I completely missed that the entities themselves do *not* need the VeluxConfigEntry. Thanks for insisting. I hope https://github.com/home-assistant/core/pull/155434/commits/bab538af169be7c68640810618fda60b8599be91 is what you actually meant :-)